app/controllers/admin/users_controller.rb

```diff
@@ -100,7 +100,16 @@ def remove_role
   private
 
   def user_params
-    params.require(:user).permit(:name, :organization_id, :email, :password, :password_confirmation)
+    user_params = params.require(:user).permit(:name, :organization_id, :email, :password, :password_confirmation)
+    role_id = updated_role_id
+
+    user_params[:organization_role_join_attributes] = { role_id: role_id } if role_id
```

Comment:
If `role_id` is nil, I feel like we should be throwing an error rather than silently swallowing it.

Reply:
Okay noted!